plugins/modules/aci_filter_entry.py

```diff
@@ -56,7 +56,7 @@
     - The Ethernet type.
     - The APIC defaults to C(unspecified) when unset during creation.
     type: str
-    choices: [ arp, fcoe, ip, mac_security, mpls_ucast, trill, unspecified ]
+    choices: [ arp, fcoe, ip, ipv6, mac_security, mpls_ucast, trill, unspecified ]
```

Comment:
Can you also add ipv4? "ip" represent both v4 and v6 so we need to add both ipv4 and ipv6 as it was added to APIC.